components/concept/menu/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     implementation Dependencies.androidx_annotation
     implementation Dependencies.androidx_appcompat
+    implementation project(':support-base')
```

Comment:
I recently ran into a similar problem and wanted to avoid having a "concept" depend on actual code here. I opened the following RFC for creating a `concept-base` component just for interfaces. That may be relevant. :) https://github.com/mozilla-mobile/android-components/pull/7776

(This doesn't need to stop landing this though)

(You primarily need the `Observable` interface here?)

Reply:
Yup, this is for the `Observable` interface.